src/components/command-bar/commands/search/unified-search/components/suggested-pages/index.tsx

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Text from 'components/text'
+import LinkRegion from 'components/link-region'
+import IconTile from 'components/icon-tile'
+import { CommandBarList } from 'components/command-bar/components'
+import { SuggestedPageProps } from './types'
+// Styles
+import s from './suggested-page.module.css'
+
+/**
+ * Renders a single suggested page list item.
+ */
+function SuggestedPage({ icon, url, text }: SuggestedPageProps) {
+	return (
+		<LinkRegion className={s.root} href={url} ariaLabel={text}>
+			<div className={s.content}>
+				<IconTile className={s.icon} size="small">
+					{icon}
+				</IconTile>
+				<div className={s.content}>
```

Comment:
❓ Are the two divs with the `s.content` class necessary here? Could we remove the `s.content` wrapper around `<Text>`?

Reply:
🙌 ah ty ty this was a silly blunder, the one around `Text` does nothing 😅 updated! ✅